packages/validation/src/permissions/has-create-research-output-permission.ts

```diff
@@ -0,0 +1,14 @@
+import { User } from '@asap-hub/auth';
+import { ResearchOutputPostRequest } from '@asap-hub/model';
+
+export const hasCreateResearchOutputPermissions = (
+  user: Omit<User, 'algoliaApiKey'>,
+  teams: ResearchOutputPostRequest['teams'],
```

Comment:
I wonder if this is the right interface, I feel like such validation method should take the user and the whole research output input, even though right now only teams are needed. Normally resource based permissions would be validated on per user and resource ID basis which is why I think that is what the input should reflect here

Reply:
On the FE we have only the team ID when we want to check if the user has a permission to show the create RO button